test_rstcheck.py

```diff
@@ -53,6 +53,9 @@ def test_check(self):
 .. code-block:: python
 
     print(
+
```

Comment:
Not sure why blank spaces were added here. It looks like the belong in `test_check_with_extra_blank_lines()`.